platform/ios/src/MGLMapView.mm

```diff
@@ -3433,6 +3433,22 @@ - (void)removeAnnotations:(NS_ARRAY_OF(id <MGLAnnotation>) *)annotations
     }
 }
 
+- (nullable NS_ARRAY_OF(id <MGLOverlay>) *)overlays
+{
+    if (self.annotations == nil) { return nil; }
+
+    NS_MUTABLE_ARRAY_OF(id <MGLOverlay>) *mutableOverlays = [NSMutableArray new];
```

Comment:
Nit: use `+[NSMutableArray array]` instead of `+[NSMutableArray new]`. In general, we avoid `+new` because it’s inconsistent with the normal init pattern.